webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageConstants.js

```diff
@@ -0,0 +1,8 @@
+import URI from 'urijs';
+import { foremanUrl } from '../../../../foreman_tools';
+
+export const docUrl = () => {
+  const rootUrl = "https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html"
```

Comment:
Don't you want to redirect to the correct section (including the `#registering-a-host-to-project-using-the-global-registration-template_managing-hosts`)

Reply:
Well that section changed name (again), so I've been thinking that maybe it is good idea to go just to page and let user scroll to correct section.

Reply:
But if I'm thinking about it again, chances that we rename registration section again are pretty low. I'll add the section back to link

Reply:
We will need to distinguish this link from any other links. I would suggest passing a parameter that will identify the section, even if we won't use it currently for link generation